src/audio/Kconfig

```diff
@@ -44,6 +44,12 @@ config COMP_MIXER
 	help
 	  Select for Mixer component
 
+config COMP_MODULE
+	bool "Loadable module component"
+	default y
+	help
+	  Select for Module component
```

Comment:
Please give more verbose `help`. I'm not sure what I'm enabling here - capability to load modules? One or more? Static/dynamic?

Reply:
I'll leave it as-is for now. This "module" component supports replacing functionality at runtime. While it was intended to be dynamic at the beginning it actually supports both static and dynamic modules. Any suggestions? (every module component can bind to one module at a time)

Reply:
Added now.